priv/test_files/unused_callbacks/warnings.erl

```diff
@@ -0,0 +1,15 @@
+-module(warnings).
+
+-callback unused_callback(map()) -> boolean().
+-callback used_callback(map()) -> boolean().
+
+-export([unused_callback/1, used_callback/1, call_callback/1]).
+
+unused_callback(#{module := Module, state := State}) ->
```

Comment:
it's used here I think

Reply:
The function name is unfortunate… but it's a great test since Hank shouldn't consider a function with the same name as a callback as usage. The callback is definitely *not* used here.